ports/air/port.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": 2,
+  "name": "air.zip",
+  "items": [
+    "Air.sh",
+    "air/"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Air",
+    "porter": [
+      "Jeod",
+	  "kloptops"
+    ],
+    "desc": "'AIR' follows Yukito Kunisaki, a traveling puppeteer searching for a winged girl from a childhood legend, who arrives in a small seaside town. There, he meets various girls, each with their own unique stories and supernatural abilities. As Yukito unravels their pasts and the town's mysteries, he discovers connections that transcend time and space, leading to poignant revelations about love, loss, and the human spirit.",
+    "inst": "Add your game assets to `ports/air/gamedata`.",
+    "genres": [
+      "visual novel"
+    ],
+    "image": "screenshot.png",
+    "rtr": false,
+    "exp": false,
+    "runtime": "rlvm.squashfs",
+	"arch": [
+		"aarch64"
+	],
+    "reqs": [["4:3"]]
```

Comment:
Does this really only work on 4:3 devices?

Reply:
Widescreen devices cannot bring up the menu to save and load games for air or kanon.